src/base/solvers/configEnvVars.m

```diff
@@ -186,7 +186,8 @@ function configEnvVars(printLevel)
 
         % check for 32-bit
         if ~isempty(strfind(computer('arch'), '64'))
-            subDir = [filesep, osPath, filesep, 'matlab'];
+            subDir = [filesep, 'matlab'];
+            %subDir = [filesep, osPath, filesep, 'matlab'];
         end
```

Comment:
Why was this changed? the default installation directory for gurobi is gurobidir\osPath\matlab at least till gurobi 7.0 ( see e.g. the (gurobi readme)[http://packages.gurobi.com/7.0/README.txt].
This is a potential reason, why so many of the tests fail in this pr.